auth/proxy/feature-flags.ts

```diff
@@ -2,6 +2,7 @@ export interface FeatureFlags {
     ATTESTATION: boolean;
 }
 
+// eslint-disable-next-line @typescript-eslint/naming-convention
 export const FEATURE_FLAGS: FeatureFlags = {
```

Comment:
The naming convention is forcing this to be camel case - however, given it's a global variable, I think `UPPER_CASING` makes the most sense?